backend/app/routers/datasets.py

```diff
@@ -372,13 +383,27 @@ async def patch_dataset(
             # Update entry to the dataset index
             doc = {
                 "doc": {
-                    "name": ds["name"],
-                    "description": ds["description"],
+                    "name": dataset["name"],
+                    "description": dataset["description"],
                     "author": UserOut(**user).email,
-                    "modified": ds["modified"],
+                    "modified": dataset["modified"],
                 }
             }
             update_record(es, "dataset", doc, dataset_id)
+            # updating metadata in elasticsearch
```

Comment:
I think this code block is repeated a couple of times in a couple of files. Would it make sense to create a function in a common `util.py` module?

Reply:
Yes, the exact code is repeated twice in datasets.py

I was thinking about creating a common function but then realized the doc would be different everywhere, hence thought of keeping it like this. If you think it would be needed at more places, I can create an utility function.